docker/Dockerfile

```diff
@@ -1,8 +1,10 @@
+ARG dev
 FROM python:3.7.2-alpine3.8
```

Comment:
The build is currently failing. I think it's because in `docker-compose.yml` under `web` you specify `target` as `prod` but this is not defined. I think on this line you need to add `FROM python:3.7.2-alpine3.8 AS prod`?

Reply:
I initially introduced build targets but for this stage we don't need them as the behavior of installing/not installing dev dependencies can be achieved with a focused build argument.